tests/snapshots/help__ping.snap

```diff
@@ -105,4 +106,22 @@ Telemetry for Orogene is opt-in, anonymous, and is used to help the team improve
 
 Sentry DSN (access token) where telemetry will be sent (if enabled)
 
+#### `--proxy`
+
+Use proxy to delegate the network.
+
+Proxy is opt-in, it uses for outgoing http/https request. If enabled, should set proxy-url too.
+
+#### `--proxy-url <PROXY_URL>`
+
+A proxy to use for outgoing http requests
+
+#### `--no-proxy-domain <NO_PROXY_DOMAIN>`
+
+Domain extensions that should bypass any proxies.
+
+using `--no-proxy-domain` to set. For example: ".host1.com, ".host2.com" ，or through environment variable "NO_PROXY" to setsuch as "NO_PROXY=.host1.com, ".host2.com"
+
+\[default: NO_PROXY]
+
```

Comment:
You need to regenerate these.

Reply:
Thanks again. I didn't notice it at all.